Modules/admin/firmware_view.php

```diff
@@ -1,17 +1,11 @@
-<?php global $path; ?>
-<link rel="stylesheet" href="<?php echo $path?>Modules/admin/static/admin_styles.css?v=<?php echo $v ?>">
+<link rel="stylesheet" href="<?php echo $path?>Modules/admin/static/admin_styles.css">
 <style>
-.content-container {
-  max-width:1150px;
-}
-
 #emonhub-running-notice, #emonhub-stopped-notice {
   padding: 8px 8px 8px 14px;
   line-height:31px;
 }
-
 </style>
-
+<div class="admin-container" style="margin-top:10px">
```

Comment:
I can't find css class admin-container in a css file but there are several uses where it sets the margin-top:10px its would be good to add the style into admin_styles.css file.